src/main/api/deleteOAuthScope.json

```diff
@@ -0,0 +1,35 @@
+{
+  "uri": "/api/application",
+  "comments": [
+    "Hard deletes a custom OAuth scope.",
```

Comment:
Since there is no soft delete, we should probably change this and the deleteApplicationRole comment to just say

`Deletes a custom OAuth scope` or `Deletes an application role`.